docs/advanced/payments/existing-domain-events.mdx

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 5.4
+title: Existing Domain Events
+description: See the detailed list of existing Domain Events.
+---
+
+# Payment events
+
+Payment events allow to inform payment event listeners about a payment status
+change.
+
+Event listeners can use this information to maintain a consistent state in a
+remote system. For instance, an eCommerce integration will implement an Order
+event listeners responsible for updating the order status accordingly on a
+headless eCommerce platform.
+
+Note: each event is based on a `PurchaseIdentifier` allowing Front-Commerce to
+retrieve the related order based on either an orderId or the cartId that was
+used before the order creation.
+
+## The paymentData attribute
+
+Most events allows you to provide custom, optional and arbitrary `paymentData`
+values. These values may contain data specific to the payment platform. We
+recommend that event listeners push these information to remote systems as they
+usually can be helpful for further processing (e.g: synchronizing a headless
+eCommerce platform with an ERP)
+
+Example: Front-Commerce Magento's integration, will append this `paymentData` as
+a JSON payload to the order history comment message.
+
+## PaymentAuthenticationStarted
+
+The authentication of the payment started (e.g. 3D Secure control step)
+
+```js
+import PaymentAuthenticationStarted from "server/modules/front-commerce/payment/domain/events/PaymentCaptureStarted";
+
+return [new PaymentAuthenticationStarted(new PurchaseIdentifier({ orderId }, pspReference, authenticationMethodName, paymentData)),
+```
+
+- `pspReference` : the reference of the transaction on the payment provider side
+- `authenticationMethodName` : optional, provides the authentication method used
+  by the payment provider in the order history
```

Comment:
in addition to the comment just above, that should be in the code (that's already the case for `authenticationMethodName`)